addons/grenades/functions/fnc_addChangeFuseItemContextMenuOptions.sqf

```diff
@@ -33,8 +33,9 @@
             }
         ],
         {
-            params ["_unit", "", "", "", "_magArr"];
-            _unit addMagazine (_magArr select 1);
+            params ["_unit", "", "", "_slot", "_magArr"];
+            private _containerStr = toLower (_slot splitString "_" select 0);
```

Comment:
What if the name already contains a _ ?

Reply:
It can only return these though can't it? https://github.com/CBATeam/CBA_A3/blob/master/addons/ui/fnc_addItemContextMenuOption.sqf#L20-L75

Reply:
Less terrible method implemented